src/addon-manager.js

```diff
@@ -1095,36 +1096,42 @@ class AddonManager extends EventEmitter {
    * @returns A promise which is resolved when updating is complete.
    */
   async updateAddons() {
+    const url = config.get('addonManager.listUrl');
     const api = config.get('addonManager.api');
     const architecture = Platform.getArchitecture();
+    const version = pkg.version;
+    const nodeVersion = Platform.getNodeVersion();
+    const pythonVersions = Platform.getPythonVersions();
     const addonPath = UserProfile.addonsDir;
     const available = {};
 
     console.log('Checking for add-on updates...');
 
     try {
-      const response = await fetch(
-        config.get('addonManager.listUrl'),
-        {headers: {'User-Agent': `mozilla-iot-gateway/${pkg.version}`}});
+      const params = new URLSearchParams();
+      params.set('api', api);
+      params.set('arch', architecture);
+      params.set('version', version);
+      params.set('node', nodeVersion);
```

Comment:
Should this also set the `test` param?

Reply:
Nope, I intentionally left that out here.

1. In test mode, this path is skipped anyway.
2. If someone manually sets the flag in normal (non-test) mode, it probably means they're (we're) testing some stuff, so we shouldn't overwrite test add-ons.